cumulusci/robotframework/Salesforce.py

```diff
@@ -333,10 +270,11 @@ def _populate_field(self, locator, value):
         field.clear()
         field.send_keys(value)
 
+    @selenium_retry
     def populate_form(self, **kwargs):
         for name, value in kwargs.items():
             locator = lex_locators["object"]["field"].format(name)
-            self._call_selenium("_populate_field", True, locator, value)
+            self._populate_field(locator, value)
```

Comment:
Yeah, I think we should move the decorator to _populate_field and remove it here, so that we retry filling specific fields rather than the entire form.